### PR TITLE
`tools/generator-go-sdk` - fix package name typo

### PR DESCRIPTION
**Pull request by catriona-m**

### CODE REVIEW

File: tools/generator-go-sdk/internal/generator/templater_id_parser.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ import (
     "strings"
 
     "github.com/hashicorp/go-azure-helpers/resourcemanager/resourceids"
-    "github.com/hashicorp/go-azure-helpers/resourcemanager/resourceids/recaser"
+    "github.com/hashicorp/go-azure-helpers/resourcemanager/recaser"
```
Comment (tombuildsstuff) [inline tools/generator-go-sdk/internal/generator/templater_id_parser.go:44]: _minor_ mind fixing the ordering here?

**Review by tombuildsstuff — COMMENTED** [review]
The ordering wants fixing but otherwise 👍

**Review by tombuildsstuff — APPROVED** [review]
Forgot to also hit approve

File: tools/generator-go-sdk/internal/generator/templater_id_parser.go (hunk 1 above)
Comment (catriona-m) [reply tools/generator-go-sdk/internal/generator/templater_id_parser.go]: Hadn't noticed that - fixed now! Thanks!

**Pull request merged**